pkg/cmd/attestation/verification/attestation.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/sigstore/sigstore-go/pkg/bundle"
 )
 
+const SLSAPredicateType = "https://slsa.dev/provenance/v1"
```

Comment:
no strong opinion but what if:

```suggestion
const SLSAPredicateV1 = "https://slsa.dev/provenance/v1"
```

Reply:
Agree here on having some versioning.